source/wp-content/themes/wporg-developer-2023/src/style/style.scss

```diff
@@ -32,17 +32,17 @@ pre {
 		padding: 20px;
 		background-color: #f7f7f7;
 		border: 1px solid var(--wp--preset--color--light-grey-1);
-		border-radius: 2px;
+		border-radius: var(--wp--custom--button--border--radius);
```

Comment:
While this is the same value, semantically it's a different use, and I think we should add a new custom var
```suggestion
		border-radius: var(--wp--custom--code--border--radius);
```

Reply:
Are you suggesting adding style in the parent theme [here](https://github.com/WordPress/wporg-parent-2021/blob/trunk/source/wp-content/themes/wporg-parent-2021/theme.json#L142-L146)? If so, do you think we should also include the complete styling for the code here, like the color of the code, padding, etc.? Perhaps we can ask the design team to provide a standard design for the code.

And what do you think we should call [this](https://github.com/WordPress/wporg-developer/blob/16766c3e87ceed66393f5f84ff4a31f442f74801/source/wp-content/themes/wporg-developer-2023/src/style/style.scss#L636)? I reckon a callout is not a typical button or code?

```
"code": {
	"border": {
		"color": "var(--wp--preset--color--light-grey-1)",
		"radius": "2px",
		"style": "solid",
		"width": "1px"
	}
},
```

Or you're just referring to adding something like [these two](https://github.com/WordPress/wporg-developer/blob/16766c3e87ceed66393f5f84ff4a31f442f74801/source/wp-content/themes/wporg-developer-2023/src/style/style.scss#L629-L630).

Reply:
```javascript
"code": {
	"border": {
		"radius": "2px",
	}
},
``` 

Yep exactly that. I guess it makes sense to add it to parent, even if most child themes might not use it.

Reply:
> And what do you think we should call [this](https://github.com/WordPress/wporg-developer/blob/16766c3e87ceed66393f5f84ff4a31f442f74801/source/wp-content/themes/wporg-developer-2023/src/style/style.scss#L636)? I reckon a callout is not a typical button or code?

Callouts are definitely different, not sure what you're asking here sorry... can you expand on it?

Reply:
Ah, yeah. I'm referring to the `button` in the `--wp--custom--button--border--radius`.
Since this border radius belongs to a callout, I assume we also want it to be something like `--wp--custom--callout--border--radius`.

Reply:
The custom code block style has been added to [parent](https://github.com/WordPress/wporg-parent-2021/pull/125). And the button has also been replaced with code here through 39fb3f1.

## Results
![image](https://github.com/WordPress/wporg-developer/assets/18050944/ce439be0-141d-4dac-9026-73f9735f440d)
![image](https://github.com/WordPress/wporg-developer/assets/18050944/d4490f0b-5292-410a-91b3-62c9506976fc)

Reply:
> Ah, yeah. I'm referring to the `button` in the `--wp--custom--button--border--radius`. Since this border radius belongs to a callout, I assume we also want it to be something like `--wp--custom--callout--border--radius`.

Sorry I get it now. Yeah I don't think this change should be included, unless we're going to update the [mu-plugins callout styles](url) to also use a var.